GSA/digitalgov.gov
# Bc update dap with ga4 info

## Summary

Added a new page to the DAP guide for GA4 migration as requested by our POC.
I will create another PR later to add the images and alt text. 
 
### Preview

[Link to Preview]()

### How To Test

1. Check the formatting
2. Check for spelling errors
3. Check links and new guide style

---

### Dev Checklist
